docs/index.rst

```diff
@@ -1,57 +1,33 @@
-============================
-Composable Kernel User Guide
-============================
-
-------------
-Introduction
-------------
+.. meta::
+  :description: Composable Kernel documentation and API reference library
+  :keywords: composable kernel, CK, ROCm, API, documentation
 
-This document contains instructions for installing, using, and contributing to Composable Kernel (CK).
+.. _composable-kernel:
 
------------
-Methodology
------------
+********************************************************************
+Composable Kernel User Guide
+********************************************************************
 
-Composable Kernel (CK) library aims to provide a programming model for writing performance critical
-kernels for machine learning workloads across multiple architectures including GPUs, CPUs, etc,
-through general purpose kernel languages, like HIP C++.
+The Composable Kernel (CK) library provides a programming model for writing performance critical kernels for machine learning workloads across multiple architectures including GPUs and CPUs, through general purpose kernel languages like HIP C++. This document contains instructions for installing, using, and contributing to the Composable Kernel project. To learn more see :ref:`what-is-ck`.
 
-CK utilizes two concepts to achieve performance portability and code maintainability:
+The CK documentation is structured as follows:
 
-* A tile-based programming model
-* Algorithm complexity reduction for complex ML operators, using innovative technique we call
-  "Tensor Coordinate Transformation".
+.. card:: :ref:`Working with Composable Kernel Library`
```

Comment:
Let's try to stick with the simple Diataxis headings for card titles (see here for the ROCm card titles: https://rocm.docs.amd.com/en/latest/)...and also use sentence case for all titles.

Reply:
I have edited and reordered the card titles to be more consistent.